### DISCUSSION TITLE
Gliner and IOB format

### DISCUSSION THREAD

**Discussion by tcourat** [urchade/GLiNER#70] [Q&A]
I have data with IOB format. But it seems to me that if I want to use this model, then I can drop the "beginning" and "inside" format (and only use regular class like LOC instead of B-LOC and I-LOC) as the model can predict the full span encapsulating both the "beginning" and "inside" tokens from the same entity ?

Can you confirm ?

**Comment by urchade** [comment]
Yes, the model predict the inclusive start and end position
